daemon.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+import sys, os, time, atexit
+from signal import SIGTERM
+
+class Daemon:
```

Comment:
If you want to support Python 2.x, you shouldn't use old-style classes any more.